Makefile

```diff
@@ -1,17 +1,12 @@
 .PHONY: all binary dynbinary build cross help install manpages run shell test test-docker-py test-integration test-unit validate validate-% win
 
 BUILDX_VERSION ?= v0.9.1
-
-ifdef USE_BUILDX
-BUILDX ?= $(shell command -v buildx)
-BUILDX ?= $(shell command -v docker-buildx)
-DOCKER_BUILDX_CLI_PLUGIN_PATH ?= ~/.docker/cli-plugins/docker-buildx
-BUILDX ?= $(shell if [ -x "$(DOCKER_BUILDX_CLI_PLUGIN_PATH)" ]; then echo $(DOCKER_BUILDX_CLI_PLUGIN_PATH); fi)
-endif
-
-ifndef USE_BUILDX
-DOCKER_BUILDKIT := 1
-export DOCKER_BUILDKIT
+ifneq (, $(BUILDX_BIN))
+	BUILDX := $(BUILDX_BIN)
+else ifneq (, $(shell docker buildx version))
+	BUILDX := docker buildx
+else ifneq (, $(shell which buildx))
+	BUILDX := $(which buildx)
```

Comment:
Wondering if we need this, can we just make `BUILDX` the way to override, and default it to `docker buildx` (or `docker`, once we have v23.0.0 I guess?)

Reply:
I guess we can yes as current docker cli package (20.10) install buildx plugin.